src/main/scala/viper/server/core/VerificationWorker.scala

```diff
@@ -120,12 +115,13 @@ class ViperBackend(private val _frontend: SilFrontend, private val _ast: Program
   /** Run the backend verification functionality
     * */
   def execute(args: Seq[String]): Unit = {
-    val (head, tail) = args.splitAt(args.length-1)
-    val fileless_args = head ++ Seq("--ignoreFile") ++ tail
+    // --ignoreFile is not enough as Silicon still tries to parse the provided filepath unless
+    // the following dummy file is used:
+    val argsWithDummyFilename = args ++ Seq("--ignoreFile", Silicon.dummyInputFilename)
```

Comment:
good point, I'll open a PR for Silicon in addition. Nevertheless, I would keep this PR as is as it is a second solution to the same problem and also cleans up VerificationWorker a bit to not make assumptions about the order of arguments

Reply:
I agree. Could you please add a small comment here with the a reference to that Silicon PR?

Reply:
It is [Silicon issue #552](https://github.com/viperproject/silicon/issues/552). The tests are still running and the PR will follow soonish

Reply:
Sorry, I meant adding the reference into the code comments. Just in case someone later on wonders why both things are necessary, or whether there's a further simplification.

Reply:
done